interface/orders/procedure_stats.php

```diff
@@ -264,6 +264,18 @@ function process_result_code($row) {
  .detail    { color:#000000; font-family:sans-serif; font-size:10pt; font-weight:normal }
 </style>
 <script type="text/javascript" src="../../library/textformat.js"></script>
+<script type="text/javascript" src="../../library/js/jquery-1.9.1.min.js"></script>
+<script type="text/javascript" src="../../library/report_validation.js"></script>
+
+<?php
+  call_required_libraries(array("iziModalToast"));
```

Comment:
calling jquery here and please use the latest version of the assets i think `3.3.1` if I am not mistaken

Reply:
Should I fix any other instances in other files that reference the older version of jQuery?

Reply:
Hi @mhn94 yes but if you feel like it slows you down just fix the ones with the files in this PR

Reply:
Hi @mhn94 yes but if you feel like it slows you down just fix the ones with the files in this PR